examples/extended/create.sh

```diff
@@ -5,7 +5,8 @@ kubectl.sh create -f existing_job.yaml
 
 kubectl.sh create -f ../../manifests/appcontroller.yaml
 
-sleep 10
+#wait for appcontroller pod creation
+sleep 20
```

Comment:
OR we can put a docker pull here to pull the latest version of the image. I will do just that.